### PR TITLE
Support GHC 9.0

### LINKED ISSUE #352: Support for ghc-9.0.1
* I've not found any issue or pr about, so i am not even sure if this already supports it

### PR DESCRIPTION
**Pull request by tfausak**
This will fix #352, but it's very much a work in progress.

### CODE REVIEW

**Comment by jneira** [comment]
Many thanks for take care, imagining how cumbersome is to have to keep up to date all these packages

File: stack.yaml [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+- butcher-1.3.3.2@sha256:0be5b914f648ec9c63cb88730d983602aef829a7c8c31343952e4642e6b52a84,3150
+- czipwith-1.0.1.3@sha256:24b13ff95927d29759b914bc5317adb53f3c15f1dd351092d648fbb2df4480eb,1671
+- data-tree-print-0.1.0.2@sha256:d845e99f322df70e0c06d6743bf80336f5918d5423498528beb0593a2afc1703,1620
+- multistate-0.8.0.3@sha256:49d600399f3a4bfd8c8ba2e924c6592e84915b63c52970818982baa274cd9ac3,3588
+
+# In the dependencies for butcher-1.3.3.2:
+#     base-4.15.0.0 from stack configuration does not match >=4.11 && <4.15
+# In the dependencies for czipwith-1.0.1.3:
+#     base-4.15.0.0 from stack configuration does not match >=4.11 && <4.15
+#     template-haskell-2.17.0.0 from stack configuration does not match >=2.9 && <2.17
```
[the remark this answers is not in the document]
Comment (tfausak) [reply stack.yaml]: Ah, that was easy 😄  https://github.com/lspitzner/czipwith/pull/2

**Comment by tfausak** [comment]
GHC 9.2 was just released: https://discourse.haskell.org/t/ghc-9-2-1-released/3527

**Comment by tfausak** [comment]
@jneira: Question for you about HLS integration: Does Brittany need to support all the same versions of HLS in order to be bundled with it? I would much rather support a single version of GHC and provide static binaries if that's possible.

**Comment by jneira** [comment]
I am afraid that is the case, each hls version is shipped for each supported ghc.

**Comment by tfausak** [comment]
Closing this in favor of #357.

**Pull request closed without merging**